plugins/PolylangPro.php

```diff
@@ -44,6 +44,14 @@ public function getDownloadUrl() {
 			'url'        => getenv( 'POLYLANG_PRO_URL' ),
 			'version'    => $this->version,
 		) ), true );
+
+		/*
+		 * If the response is not the version we asked for then, bail
+		 */
+		if ( $this->version != $response['stable_version']) ) {
```

Comment:
@jarstelfox Could you please make this:

`if ( $response['stable_version'] !== $this->version  ) ) {` to be in line with WP coding standards?

Reply:
```suggestion
		if (  $response['stable_version'] !== $this->version ) ) {
```